src/index.test.ts

```diff
@@ -431,4 +431,21 @@ describe('ember-template-recast', function () {
       '{{foo-bar placeholder="Choose a \\"thing\\"..." p1="Choose a \\"thing\\"..."}}'
     );
   });
+
+  test('stack trace does not exceed', function () {
```

Comment:
Let's tweak the name here:

```suggestion
  test('can replace an `AttrNode`s value with a new `ConcatStatement`', function () {
```